style.css

```diff
@@ -1,340 +1,1555 @@
+    line-height: 56px;
+}
+
+.intro__description {
+    margin-bottom: 30px;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+}
+
+/* Index page Catalog block */
+
+.catalog {
+    position: relative;
+    z-index: 2;
+}
+
+.catalog__navigation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 30px;
+    gap: 30px;
+}
+
+.catalog__wrapper {
+    /* 
+    display: grid;
+    grid-template-columns: repeat(5, 19%);
+    grid-auto-flow: column;
+    */
+    display: flex;
+    gap: 32px;
+    overflow-x: hidden;
+}
+
+.catalog__actions {
+    position: relative;
+    display: flex;
+    order: 3;
+    column-gap: 16px;
+}
+
+.catalog__actions:after {
+    content: '';
+    position: absolute;
+    top: 10px;
+    right: -20px;
+    width: 2px;
+    height: 24px;
+    background-color: #D0D7FB2B;
+}
+
+.catalog__title {
+    position: relative;
+    padding-left: 30px;
+    font-family: var(--font-secondary);
+    font-size: 24px;
+    line-height: 32px;
+    order: -1;
+}
+
+.catalog__title::before {
+    position: absolute;
+    content: '';
+    top: 3px;
+    left: 0px;
+    width: 24px;
+    height: 24px;
+    background-image: url('images/folder.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+.catalog__navigation:before {
+    content: '';
+    background: #D0D7FB2B;
+    height: 1px;
+    flex: 1 1 auto;
+}
+
+.catalog__item {
+    position: relative;
+    flex-basis: 20%;
+    padding: 30px;
+    background-color: #1D1C28;
+    transition: all 1s ease;
+}
+
+.catalog__item:hover {
+    background-image: url(images/hover-up-left.webp), url(images/hover-down-right.webp);            
+    background-repeat: no-repeat, no-repeat;
+    background-position: top left, bottom right;
+}
+
+.catalog__item-overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 40px;
+    box-sizing: border-box;
+    background-color: rgba(29, 28, 40, 0.5);
+}
+
+.catalog__item:hover .catalog__item-overlay {
+    transition: all .5s ease-in-out;
+    opacity: 1;
+}
+
+.catalog__btns {
+    position: absolute;
+    width: 100%;
+    padding: 0 30px;
+    box-sizing: border-box;
+}
+
+.catalog__btns > .btn + .btn {
+    margin: 24px 0 0 0;
+}
+
+.catalog__item:hover > .catalog__overlay {
+    display: block;
+}
+
+.catalog__image {
+    min-width: 240px;
+    position: relative;
+}
+
+.catalog__price {
+    font-family: var(--font-secondary);
+    font-size: 36px;
+    line-height: 48px;
+    letter-spacing: -0.5px;
+}
+
+.catalog__name {
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+    color: var(--color-text);
+}
+.catalog--big {
+    display: flex;
+    flex-direction: column;
+    row-gap: 32px;
+}
+
+.catalog--big .catalog__wrapper {
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
1. .catalog--big .catalog__wrapper Would be better us modificator instead parent selector. For example: class="catalog__wrapper catalog__wrapper--columns"
2. Delete display: flex; Duplicated on the line 305